include/matx/core/sparse_tensor_format.h

```diff
@@ -255,42 +255,48 @@ template <int D, typename... LvlSpecs> class SparseTensorFormat {
   template <typename CRD, int L = 0>
   __MATX_INLINE__ __MATX_HOST__ __MATX_DEVICE__ static void
   dim2lvl(const CRD *dims, CRD *lvls, bool asSize) {
-    using ftype = std::tuple_element_t<L, LVLSPECS>;
-    if constexpr (ftype::expr::op == LvlOp::Id) {
-      lvls[L] = dims[ftype::expr::di];
-    } else if constexpr (ftype::expr::op == LvlOp::Div) {
-      lvls[L] = dims[ftype::expr::di] / ftype::expr::cj;
-    } else if constexpr (ftype::expr::op == LvlOp::Mod) {
-      lvls[L] =
-          asSize ? ftype::expr::cj : (dims[ftype::expr::di] % ftype::expr::cj);
-    }
-    if constexpr (L + 1 < LVL) {
-      dim2lvl<CRD, L + 1>(dims, lvls, asSize);
+    if constexpr (L < LVL) {
```

Comment:
These changes look like the same ones from the other PR?

Reply:
Yeah (I am merging them as the other PRs go in, but they were all required to keep the example running properly)